test/unit/Cover/initialize.js

```diff
@@ -0,0 +1,30 @@
+const { expectRevert } = require('@openzeppelin/test-helpers');
+
+const { bnEqual } = require('../utils').helpers;
```

Comment:
:warning: Please stop using oz test helpers and bnEqual for v2!  
- `bnEqual` raises a warning with instructions on what to use instead.
- `expectRevert` should be replaced with chai assertions (`to.be.reverted` thing)